### ISSUE TITLE
Incorrect documentation for compound IDs in Authfile

### ISSUE THREAD

**Issue by bbockelm** [xrootd/xrootd#2061]
The posted grammar for compound IDs in the documentation is incorrect as well as the example.  Here's the example from the documentation:

```
= atlddm  o atlas r production u ddm
= atlprod o atlas r production
x atlddm  /atlas ld
x atlprod /atlas lrw
```

However, this is invalid syntax for compound rules; if you copy/paste this into your config, XRootD will refuse to start.

Instead, you need:

```
= atlddm  o: atlas r: production u: ddm
= atlprod o: atlas r: production
x atlddm  /atlas ld
x atlprod /atlas lrw
```

This is the relevant code:

https://github.com/xrootd/xrootd/blob/master/src/XrdAcc/XrdAccAuthFile.cc#L160-L167

It's unclear to me why this extra character is necessary.  In fact, there's no check for the semicolon.  Despite the code comment indicating that `c:` is the correct format, it only checks there are two characters.  Something like `cc` or `c!` appears to pass validation.

**Comment by abh3** [comment]
So, the question is whether to change the documentation or to change the code. The particular problem here is that the test should have been strlen(id) != 1 as opposed to '2'. The comment was a hold over after the following attempted fix:
11 November  2020
     Correct documentation on how to specify the idtype (i.e. the single letter is not followed by a colon).

So, to keep things sane. The code will change to secretly allow two characters but only consider the first. That makes it backward compatible without needing the change the documentation as it was already changed but the code was not properly changed. Must have been a pandemic driven change, sigh.

**Comment by amadio** [comment]
Fixed by 478ad4b4c4ce305d072249d7f25c5aa134d44f69.
